### PR TITLE
Added RelativeCenterRD to the ConfigurationFile.

### PR DESCRIPTION
**Pull request by bartburkhardt**
ReferenceWGS84 is now derived from the RelativeCenterRD setting.
Removed the setters for UnitsPerDegreeX and UnitsPerDegreeY to reduce complexity
Added LocationSuggestionUrl to the ConfigurationFile

### CODE REVIEW

File: 3DAmsterdam/Assets/Netherlands3D/Scripts/Utilities/ConvertCoordinates.cs [hunk 1]
```diff
@@ -45,102 +63,52 @@ public Vector3WGS(double Lon, double Lat, double H)
...
-                //update referenceWGS84
-                Vector3WGS refWGS = RDtoWGS84(referenceRD.x, referenceRD.y);
-                double hoogte = RDCorrection(refWGS.lon, refWGS.lat, "Z", RDCorrectionZ);
-                referenceWGS84 = new Vector3WGS(refWGS.lon, refWGS.lat,hoogte);
-                //update unitsperdegreeX
-                unitsPerDegreeX = Math.Cos(referenceWGS84.lon * Math.PI / 180) * unitsPerDegreeY;
+                if(_referenceWGS84 == null)
+                {
+                    _referenceWGS84 = RDtoWGS84(Config.activeConfiguration.RelativeCenterRD.x, Config.activeConfiguration.RelativeCenterRD.y);                    
+                }
```
Comment (mvangog) [inline 3DAmsterdam/Assets/Netherlands3D/Scripts/Utilities/ConvertCoordinates.cs:88]: should also calculate the wgs-height (like in old lines 102-103). wgs-hegiht is not rd-height

File: 3DAmsterdam/Assets/Netherlands3D/ScriptableObjects/ConfigurationFile.cs [hunk 2]
```diff
@@ -16,6 +17,9 @@ public enum TmsTileNumberingType
         }
 
         [Header("Coordinates")]
+
+        public Vector2RD RelativeCenterRD;
+
         public float zeroGroundLevelY = 43.0f;
```
Comment (mvangog) [inline 3DAmsterdam/Assets/Netherlands3D/ScriptableObjects/ConfigurationFile.cs:23]: this is a bit confusing now:
  relativeCenterRD are the RDcoordinates at unity-Origin
  zeroGroundLevelY is the Unity-Y-axis value of the NAP-plane

wouldn't it be better if we have a Vector3RD for RdAtUnityORigin?

Comment (mvangog) [reply 3DAmsterdam/Assets/Netherlands3D/ScriptableObjects/ConfigurationFile.cs:23]: or a float for NAP-elevation of the UnityOrigin?

File: 3DAmsterdam/Assets/Netherlands3D/Scripts/Utilities/ConvertCoordinates.cs [hunk 3]
```diff
@@ -45,102 +63,52 @@ public Vector3WGS(double Lon, double Lat, double H)
...
         private static double unitsPerDegreeY = 111000; //approximation of distance between lattitudinal degrees in meters
 
-
         private static byte[] RDCorrectionX = Resources.Load<TextAsset>("x2c").bytes;
         private static byte[] RDCorrectionY = Resources.Load<TextAsset>("y2c").bytes;
         private static byte[] RDCorrectionZ = Resources.Load<TextAsset>("nlgeo04").bytes;
 
         private static Vector3RD output = new Vector3RD();
+
+        private static Vector3WGS? _referenceWGS84  = null;
```
[the remark this answers is not in the document]
Comment (bartburkhardt) [reply 3DAmsterdam/Assets/Netherlands3D/Scripts/Utilities/ConvertCoordinates.cs]: I have changed this

File: 3DAmsterdam/Assets/Netherlands3D/Scripts/Utilities/ConvertCoordinates.cs (hunk 1 above)
Comment (bartburkhardt) [reply 3DAmsterdam/Assets/Netherlands3D/Scripts/Utilities/ConvertCoordinates.cs:94]: Thanks, I made an update

**Pull request merged**